sdc/tests/test_dataframe.py

```diff
@@ -1150,6 +1153,37 @@ def test_impl(n):
         hpat_func = self.jit(test_impl)
         pd.testing.assert_series_equal(hpat_func(n), test_impl(n))
 
+    def test_dataframe_head(self):
+        def test_impl(df):
+            return df.head()
+        sdc_func = sdc.jit(test_impl)
+        df = pd.DataFrame({"FLOAT": test_global_input_data_float64[0][:5],
+                           "DATATIME": test_datatime,
+                           "INT": test_global_input_data_int64[:5],
+                           "STRING": ['a', 'dd', 'c', '12', 'ddf']})
+        pd.testing.assert_frame_equal(sdc_func(df), test_impl(df))
+
+    def test_dataframe_head1(self):
```

Comment:
What if Index is set explicitly, this solution is ok for that case? I don't see tests on it